inst/tests/tests.Rraw

```diff
@@ -17261,3 +17261,11 @@ if (identical(x, enc2native(x))) {
 
 # fintersect now preserves order of first argument like intersect, #4716
 test(2163, fintersect(data.table(x=c("b", "c", "a")), data.table(x=c("a","c")))$x, c("c", "a"))
+
+# setDF now drops index attributes, #4889
+d <- data.table(a=1:100, b=1:100)
+invisible(d[a == 50])   
```

Comment:
I think the test will be clearer to future readers if we run `setindex(d, a)`

Reply:
You're right, fixing